website/docs/api/top-level.md

```diff
@@ -1004,6 +1004,56 @@ This method was previously available as `spacy.gold.spans_from_biluo_tags`.
 | `tags`      | A sequence of [BILUO](/usage/linguistic-features#accessing-ner) tags with each tag describing one token. Each tag string will be of the form of either `""`, `"O"` or `"{action}-{label}"`, where action is one of `"B"`, `"I"`, `"L"`, `"U"`. ~~List[str]~~ |
 | **RETURNS** | A sequence of `Span` objects with added entity labels. ~~List[Span]~~                                                                                                                                                                                        |
 
+### training.biluo_to_iob {#biluo_to_iob tag="function"}
+
+Convert a sequence of [BILUO](/usage/linguistic-features#accessing-ner) tags to
+[IOB](/usage/linguistic-features#accessing-ner) tags. This is useful if you want
+use the [BILUO](/usage/linguistic-features#accessing-ner) tags with a model that
+only supports [IOB](/usage/linguistic-features#accessing-ner) tags.
```

Comment:
Is there a reason you have the same link in four different places? I see that the place you're linking to in the docs explains the tag schemas, but it seems confusing to link to it repeatedly like this. I think it's enough to link to it once, and then you also don't need the link in the parameter table.

Reply:
I've pushed a commit to only keep the link for the first mentions of "IOB" and "BILUO" in the text part. However, since the precedent function docs(e.g., [biluo_tags_to_offsets](https://spacy.io/api/top-level#biluo_tags_to_offsets)) also have links to the tag schemes in the parameter table, I think we should keep the links in the parameter table to follow this pattern.

Reply:
Thanks, I think that's a lot cleaner!